qiskit/circuit/library/data_preparation/pauli_feature_map.py

```diff
@@ -281,6 +284,40 @@ def cx_chain(circuit, inverse=False):
         basis_change(evo, inverse=True)
         return evo
 
+    def get_entangler_map(
+        self, rep_num: int, block_num: int, num_block_qubits: int
+    ) -> Sequence[Sequence[int]]:
+
+        # entanglement is Dict[int, List[List[int]]]
+        if isinstance(self.entanglement, dict):
+            if all(
+                isinstance(e2, (int, np.int32, np.int64))
+                for key in self.entanglement.keys()
+                for en in self.entanglement[key]
+                for e2 in en
+            ):
+                for qb, ent in self.entanglement.items():
+                    for ind, en in enumerate(ent):
+                        if len(en) != qb:
+                            raise ValueError(
+                                f"For num_qubits = {qb}, entanglement must be a "
+                                f"tuple of length {qb}. You specified {en}."
+                            )
+                        self.entanglement[qb][ind] = tuple(map(int, en))
```

Comment:
I'm not sure we need this; users should already provide integers (as per the type hint) and I think the code works with both tuples and lists.